s3/utils.bal

```diff
@@ -241,6 +241,19 @@ isolated function populateCreateObjectHeaders(map<string> requestHeaders, Object
         if (objectCreationHeaders?.expires != ()) {
             requestHeaders[EXPIRES] = <string>objectCreationHeaders?.expires;
         }
+        if (objectCreationHeaders?.contentType != ()) {
+            requestHeaders[CONTENT_TYPE] = <string>objectCreationHeaders?.contentType;
+        }
+    }
+}
+
+# Function to populate createObject optional user-defined metadata headers.
+#
+# + requestHeaders - Request headers map.
+# + userMetadataHeaders - Map containing user-defined metadata.
```

Comment:
Since this is not a public method do we need to add documentation ?

Reply:
@ayeshLK yeah its not mandatory, but users can optionally add it. So I don't see a reason why we should remove an already added documentation :)